<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MailT</class>
 <widget class="QDialog" name="MailT">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/ressource/mariage.jpg);</string>
   </property>
   <property name="title">
    <string>MailTache</string>
   </property>
   <widget class="QLineEdit" name="rcpt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>441</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>recipient</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mail_pass">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>your mail password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subject">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>591</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="placeholderText">
     <string>subject</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>491</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="placeholderText">
     <string>attachement here</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="msg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>591</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
</string>
    </property>
    <property name="placeholderText">
     <string>type your message here ...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>591</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseBtn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Browse ...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
